src/scenes/StageScene.js

```diff
@@ -107,13 +108,20 @@ export default class StageScene extends Phaser.Scene {
     this.clickSound = this.sound.add('clickSound');
     this.clickSound.volume = 0.05;
 
-    // create jump sound
-    this.jumpSound = this.sound.add('jumpSound');
+    // create UI
+    this.createUI();
```

Comment:
remove this line of code 'cause this.createUI() is declared at the bottom of create()